R/get_census_api_2.R

```diff
@@ -6,8 +6,8 @@
 #' It is used by the \code{get_census_api} function. The user should not need to call this 
 #' function directly.
 #'
-#' @param data_url URL root of the API, including the question mark, 
-#'  e.g., \code{"https://api.census.gov/data/2010/dec/sf1?"}.
```

Comment:
Please verify.

Reply:
verified, same 2020 endpoint as the sister R script.